translations/es.json

```diff
@@ -365,5 +377,8 @@
 
   "file.desc.pdf": "<h3>Accesibilidad a PDF</h3><h4>Etiquetas PDF</h4><p>Las etiquetas PDF proporcionan una representación estructurada y oculta del contenido PDF que se presenta a los lectores de pantalla. Existen solo con fines de accesibilidad y no tienen ningún efecto visible en el archivo PDF. Un archivo PDF accesible es más que etiquetas, pero un PDF sin etiquetar no se consideraría <em>accesible</em>.</p><p>Aprenda a crear archivos PDF etiquetados en WebAIM.org:<br/><a href='https://webaim.org/techniques/acrobat/' target='_blank'>Accesibilidad PDF</a></p>",
   "file.desc.doc": "<h3>Creación de documentos de Word accesibles</h3><p>Microsoft Word es el procesador de texto más utilizado en el mercado y el formato .docx es el formato de facto para documentos de texto. También se utiliza a menudo para crear archivos PDF y HTML para sitios web. Hay varias cosas que puede hacer para que sus documentos de Word sean más accesibles para las personas con discapacidades.</p><p>Obtenga más información sobre los documentos de Word accesibles en WebAIM.org:<br/><a href='https://webaim.org/techniques/word/ 'target='_blank'>Accesibilidad de Microsoft Word</a></p>",
-  "file.desc.ppt": "<h3>Creación de presentaciones accesibles</h3><p>Microsoft PowerPoint es una de las herramientas más populares para crear presentaciones de diapositivas. A menudo se usa para organizar pensamientos para una reunión o lección, para presentar puntos clave en una presentación en vivo e incluso para crear folletos.</p><p>Obtenga más información sobre cómo crear presentaciones de PowerPoint accesibles en WebAIM.org:<br/><a href='https://webaim.org/techniques/powerpoint/' target='_blank'>Accesibilidad de Microsoft PowerPoint</a></p>"
-}
+  "file.desc.ppt": "<h3>Creación de presentaciones accesibles</h3><p>Microsoft PowerPoint es una de las herramientas más populares para crear presentaciones de diapositivas. A menudo se usa para organizar pensamientos para una reunión o lección, para presentar puntos clave en una presentación en vivo e incluso para crear folletos.</p><p>Obtenga más información sobre cómo crear presentaciones de PowerPoint accesibles en WebAIM.org:<br/><a href='https://webaim.org/techniques/powerpoint/' target='_blank'>Accesibilidad de Microsoft PowerPoint</a></p>",
```

Comment:
- presentaciones de diapositivas -> presentaciones diapositivas
- pensamientos -> ideas (same spelling in English)